content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: 'Create GDS from Tinytapeout Git Templates'
+linkTitle: 'Template to GDS'
+description: 'preparing your design for submission using the TinyTapeout GitHub template'
+weight: 60
+---
+
+
+This guide will walk you through the process of preparing your design for submission using the TinyTapeout GitHub template. We’ll cover the necessary steps to convert your design into the ASIC files required for chip manufacturing and also view your files with the 3d viewer.
+If you don't already have a GitHub account, please create one before proceeding.
+
+
+## Access the GitHub Template
+
+1. **Create a GitHub Account**: If you don’t have a GitHub account, sign up for one from [here](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).
+
+2. **Navigate to the Template Repository**: 
+   - We offer two types of repositories: one for Wokwi designs and another for Verilog or other HDLs. For this guide, we'll use the [Wokwi template](https://github.com/TinyTapeout/tt08-wokwi-template).
+
+3. **Create Your Repository**:
+   - Click the ‘Use this template’ button to generate your own copy of the repository.
+
+   <img src="images/template.png" alt="Use Template Repository" width="550"/>
+<!-- ![Use Template Repository](images/template.png) -->
+
+## Enable GitHub Actions
+GitHub Actions allow automated tasks to run. They are enabled by default, but we need to enable them to publish the results
+1. **Go to Repository Settings**:
+   - Access your repository’s settings page.
+
+2. **Configure Build and Deploy Source**:
+   - Navigate to the “Pages” section and set the build and deploy source to GitHub Actions.
+
+3. **Allow Actions**:
+   - Ensure that actions are enabled to create the results page.
+
+   <img src="images/gitpages.png" alt="Enable Git Actions" width="550"/>
+<!-- ![Enable GitHub Actions](images/gitpages.png) -->
+##  Edit the `info.yaml` File
+
+1. **Adjust Wokwi ID**:
+   - For Wokwi projects, update the Wokwi ID in the `info.yaml` file to match the ID from your Wokwi project.
+   - For HDL projects, use the source files section as covered in a separate guide titled [Working with an HDL]().
+
+   <img src="images/WOKWI_Project_ID.png" alt="Wokwi ID" width="550"/>
+<!-- ![Set the Wokwi ID](images/WOKWI_Project_ID.png) -->
+
+2. **Fill Out Metadata**:
+   - Enter the title, author information, and your Discord username if applicable.
+3. **Describe Pins**:
+   - Provide descriptions for the pins to aid in testing other designs. 
+
+   <img src="images/info_yaml.png" alt="yamlinfo" width="550"/>
```

Comment:
for the image in 'decribe pins' you have VCC and VSS which is confusing as these are GPIO pins. Clock, reset, power and ground are always supplied and don't need to be specified. So please update the image with something that matches your project description like 'input a1 input a0, input b1, input b0'.